crates/ruff/src/checkers/ast/analyze/arg.rs

```diff
@@ -0,0 +1,27 @@
+use rustpython_parser::ast::{Arg, Ranged};
+
+use crate::checkers::ast::Checker;
+use crate::codes::Rule;
+use crate::rules::{flake8_builtins, pep8_naming, pycodestyle};
+
+/// Run lint rules over an [`Stmt`] syntax node.
+pub(crate) fn arg(arg: &Arg, checker: &mut Checker) {
```

Comment:
What I mostly need feedback on is whether this API and module structure is fine or if we prefer something else. Like we could do something trait-based (but it won't work for some of the other methods in this module, like `scopes`, which checks _all_ the scopes.) Or we could make these methods on `Checker` but just implement them in separate files for convenience (but then we risk method name clashes across `impl` blocks).